unlock-app/src/utils/Error.ts

```diff
@@ -83,6 +84,8 @@ const SignUp: ErrorMakers = errorsFor('SignUp')
 // Transaction errors (failed to create lock, etc.)
 const Transaction: ErrorMakers = errorsFor('Transaction')
 
+const Web3: ErrorMakers = errorsFor('Web3')
```

Comment:
I had previously forgotten to include a category of errors for this, so I threw them in here